Fix:

error: call to undeclared library function 'exit' with type 'void (int) __attribute__((noreturn))'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
error: type specifier missing, defaults to 'int'; ISO C99 and later do not support implicit int [-Wimplicit-int]
--- configure.orig	2019-11-24 00:45:39.000000000 -0600
+++ configure	2025-08-22 01:28:08.000000000 -0500
@@ -917,6 +917,7 @@
 #line 918 "configure"
 #include "confdefs.h"
 
+int
 main(){return(0);}
 EOF
 if { (eval echo configure:923: \"$ac_link\") 1>&5; (eval $ac_link) 2>&5; } && test -s conftest${ac_exeext}; then
@@ -1167,6 +1168,7 @@
   cat > conftest.$ac_ext <<EOF
 #line 1169 "configure"
 #include "confdefs.h"
+#include <stdlib.h>
 #include <ctype.h>
 #define ISLOWER(c) ('a' <= (c) && (c) <= 'z')
 #define TOUPPER(c) (ISLOWER(c) ? 'A' + ((c) - 'a') : (c))
@@ -3227,7 +3229,7 @@
 #include "confdefs.h"
 
 int main() {
-} $ac_kw foo() {
+} $ac_kw int foo() {
 ; return 0; }
 EOF
 if { (eval echo configure:3234: \"$ac_compile\") 1>&5; (eval $ac_compile) 2>&5; }; then
